pkg/karmadactl/cmdinit/utils/util.go

```diff
@@ -21,11 +21,17 @@ type Downloader struct {
 // Read Implementation of Downloader
 func (d *Downloader) Read(p []byte) (n int, err error) {
 	n, err = d.Reader.Read(p)
+	if err != nil {
+		if err != io.EOF {
+			return
+		}
+		// The width of the log shall not less than `Downloading...[ %.2f%% ]`
+		fmt.Println("\rDownload complete.       ")
```

Comment:
:) 

It'd be nice to keep it simple, like:
```suggestion
		fmt.Println("\nDownload complete.")
```